lib/kubernetes-deploy/runner.rb

```diff
@@ -112,6 +112,21 @@ def template_variables
 
     private
 
+    # Inspect the file that was mentioned in the kubectl output
+    # to make it easier for developer to understand what's going on
+    def inspect_kubectl_out_for_yaml(out)
+      # Output example:
+      # error: unable to decode "/tmp/path/to/file": [pos 96]: json: expect char '"' but got char '{'
+      if match = out.match(/error:(.+)"(?<path>\S+)":+/)
+        path = match[:path]
+        if path.present? && File.file?(path)
```

Comment:
Good call on checking if it's actually a file :) You may want to log that we _tried_ to look for a file, but it wasn't there.